blockstore/apps/bundles/storage.py

```diff
@@ -154,8 +154,8 @@ def get_modified_time(self, name):
     def get_valid_name(self, name):
         return default_storage.get_valid_name(name)
 
-    def get_alternative_name(self, name):
-        return default_storage.get_alternative_name(name)
+    def get_alternative_name(self, file_root, file_ext):
+        return default_storage.get_alternative_name(file_root, file_ext)
```

Comment:
A parameter was added in Django 3. Though this method does not seem to be in use in this package.